x-pack/plugins/osquery/server/routes/fleet_wrapper/get_agent_details.ts

```diff
@@ -17,10 +16,20 @@ export const getAgentDetailsRoute = (router: IRouter, osqueryContext: OsqueryApp
       validate: {
         params: schema.object({}, { unknowns: 'allow' }),
       },
-      options: { tags: [`access:${PLUGIN_ID}-read`] },
     },
     async (context, request, response) => {
       let agent;
+      // this is to skip validation eg. for analysts in cases attachments so they can see the results despite not having permissions
+      const { isSystemRequest } = request;
+      if (!isSystemRequest) {
```

Comment:
Hey! Don't forget to remove this logic from all routes.

Reply:
👍